app/_hub/kong-inc/mtls-auth/index.md

```diff
@@ -162,7 +164,27 @@ $ curl -sX POST https://kong:8001/ca_certificates -F cert=@cert.pem
   "id": "322dce96-d434-4e0d-9038-311b3520f0a3"
 }
 ```
+{% endnavtab %}
+
+{% navtab Konnect Cloud %}
+
+Go through the Konnect Manager:
+1. From the Dashboard, select **Shared Config** in the left navigation 
+2. Select **Certificates** 
+3. Click **New Certificate**
+4. Copy and paste your certificate information and select **Create**
+
+You will now see your certificate listed in the **Certificates** tab.
 
+To add a certificate via curl, you are required to have:
+1. Konnect ID
+2. A generated access cookie
+
+```bash
+$ curl -sX POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```

Comment:
```suggestion
curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```

Removed `$` per our style guide. I haven't seen `-sX` before, did you mean `-X`? If not, I can revert it.

Reply:
I was just following the original docs example which has `-sX` as part of the POST command. I don't think the `-s` for quiet mode is necessary for functionality so can be removed if needed.

Reply:
```suggestion
$ curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```